dotcom-rendering/src/web/components/KeyEventCard.tsx

```diff
@@ -43,6 +43,12 @@ const linkStyles = (palette: Palette) => css`
 			border-bottom: 1px solid ${palette.hover.keyEventLink};
 		}
 	}
+
+	&:hover > div {
```

Comment:
We can move this into the `&:hover` selector above, how the span is being selected, to make the code a little cleaner